packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -65,138 +85,275 @@ private LocalAuthPlugin(Activity activity) {
    */
   public LocalAuthPlugin() {}
 
+  private LocalAuthPlugin(Registrar registrar) {
+    this.registrar = registrar;
+    this.registrar.addActivityResultListener(resultListener);
```

Comment:
There's no need for this constructor.  You can do all this in the registerWith() call.

Reply:
Moved code to `registerWith()` method and deleted extra constructor.